pkg/kubelet/server/streaming/server.go

```diff
@@ -89,7 +89,7 @@ type Config struct {
 var DefaultConfig = Config{
 	StreamIdleTimeout:     4 * time.Hour,
 	StreamCreationTimeout: remotecommand.DefaultStreamCreationTimeout,
-	SupportedProtocols:    remotecommand.SupportedStreamingProtocols,
+	SupportedProtocols:    append(remotecommand.SupportedStreamingProtocols, portforward.SupportedProtocols...),
```

Comment:
is this correct to merge the protocols for remotecommand and portforward? Would have expected to have two separate lists.